web-app/client/src/components/AccountSettings.js

```diff
@@ -121,6 +121,16 @@ const AccountSettings = ({
             </InputGroup>
           </div>
         </Form.Group>
+        <Form.Group>
+          <Form.Label>{t('organizationIcos.label')}</Form.Label>
+          <Tooltip tooltipText={t('organizationIcos.tooltip')} />
+          { loggedUser && loggedUser.organizationIcos && loggedUser.organizationIcos.length > 0 ?
```

Comment:
I guess `loggedUser` is always not null (because of `Auth` compoment)
And let's move requiring `loggedUser.organizationIcos` to branch so whole component waits for it